src/carter-mc-barter-narter-is-dope.html

```diff
@@ -13,11 +13,13 @@
   </head>
 
   <body>
+    <div class="button">The Brother's Allen</div>
```

Comment:
Hmm; is "button" on a `div` the most semantically-correct class name we can use here? Seems like an `h1`, `h2`, or `h3` tag with a `title` class might be more appropriate.

eg
```suggestion
    <h1 class="main-page-title">The Brothers Allen</h1>
```

Reply:
(also - don't need to accept my suggestion, can just use whatever you want from it!)